### PR TITLE
[DO NOT MERGE] ChartboostMediation

### PR DESCRIPTION
**Pull request by helium-service**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "chartboost-mediation-android-sdk-demo"]
```
Comment (bichenwang) [inline .gitmodules:1]: @cb-jpadilla we probably didn't want the submodule copied over.

File: chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        }
+    }
+}
+
+dependencies {
+    // Chartboost Mediation SDK
+    implementation("com.chartboost:chartboost-mediation-sdk:4.9.0")
+
+    // Chartboost Mediation adapters
+    implementation("com.chartboost:chartboost-mediation-adapter-admob:4.+")
```
Comment (bichenwang) [inline chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts:57]: update to 5.+?

File: chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+    // Chartboost Mediation SDK
+    implementation("com.chartboost:chartboost-mediation-sdk:4.9.0")
```
Comment (bichenwang) [inline chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts:54]: you sure we sync'd the right branch?

File: chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts (hunk 2 above)
Comment (cb-jpadilla) [reply chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts:57]: The update work to will be done separately by @bwised.

File: chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts (hunk 3 above)
Comment (cb-jpadilla) [reply chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts:54]: Yeah. Demo app update work will be done by @bwised.

File: .gitmodules (hunk 1 above)
Comment (cb-jpadilla) [reply .gitmodules:1]: Oh, I see, will adjust the script. 📓

File: chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts (hunk 3 above)
Comment (cb-jpadilla) [reply chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts:54]: We don't need to merge this PR, but now that we know rsync can add the apps, 🤩 will push the script changes that were made to get this to run successfully.

**Comment by cb-jpadilla** [comment]
Successful run, but closing until we properly have the updated changes demo apps with 5.0.0.

**Pull request closed without merging**